server/routes/api/middlewares/registerRateLimiter.ts

```diff
@@ -0,0 +1,18 @@
+import { Context, Next } from "koa";
+import { RateLimiterRedis } from "rate-limiter-flexible";
+import { RateLimiterConfig } from "@server/types";
+
+export default function registerRateLimiter(
```

Comment:
Could put this in the file with other middleware to be honest as they are highly coupled

Reply:
Sure! Will follow it up as part of the refactor.

Reply:
Fixed!